ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm_UI.robot

```diff
@@ -0,0 +1,572 @@
+    ...    User Can Fetch Number Of Requests Over Defined Time    thanos_url=${thanos_url}    thanos_token=${token}
+    ...    model_name=${flan_model_name}    query_kind=single    namespace=${test_namespace}    period=5m    exp_value=3
+    Wait Until Keyword Succeeds    20 times    5s
+    ...    User Can Fetch Number Of Successful Requests Over Defined Time    thanos_url=${thanos_url}
+    ...    thanos_token=${token}    model_name=${flan_model_name}    namespace=${test_namespace}    period=5m
+    ...    exp_value=3
+    Wait Until Keyword Succeeds    20 times    5s
+    ...    User Can Fetch CPU Utilization    thanos_url=${thanos_url}    thanos_token=${token}
+    ...    model_name=${flan_model_name}    namespace=${test_namespace}    period=5m
+    Query Model Multiple Times    model_name=${flan_model_name}
+    ...    endpoint=${CAIKIT_STREAM_ENDPOINT_HTTP}    n_times=1    streamed_response=${TRUE}
+    ...    namespace=${test_namespace}    query_idx=${0}    protocol=http    validate_response=${FALSE}
+    Wait Until Keyword Succeeds    30 times    5s
+    ...    User Can Fetch Number Of Requests Over Defined Time    thanos_url=${thanos_url}    thanos_token=${token}
+    ...    model_name=${flan_model_name}    query_kind=stream    namespace=${test_namespace}    period=5m    exp_value=1
+    [Teardown]    Clean Up DSP Page
+
+
+*** Keywords ***
+Setup Kserve UI Test
+    [Documentation]    Instaling And Configuring dependency operators: Service Mesh and Serverless.
+    ...                This is likely going to change in the future and it will include a way to skip installation.
+    ...                Caikit runtime will be shipped Out-of-the-box and will be removed from here.
+    [Arguments]    ${user}=${TEST_USER.USERNAME}    ${pw}=${TEST_USER.PASSWORD}    ${auth}=${TEST_USER.AUTH_TYPE}
+    Set Library Search Order  SeleniumLibrary
+    Skip If Component Is Not Enabled    kserve
+    RHOSi Setup
+    Load Expected Responses
+    Launch Dashboard    ${user}    ${pw}    ${auth}    ${ODH_DASHBOARD_URL}    ${BROWSER.NAME}    ${BROWSER.OPTIONS}
+    Fetch CA Certificate If RHODS Is Self-Managed
+
+Load Expected Responses
+    [Documentation]    Loads the json file containing the expected answer for each
+    ...                query and model
+    ${exp_responses}=    Load Json File    ${EXP_RESPONSES_FILEPATH}
+    Set Suite Variable    ${EXP_RESPONSES}    ${exp_responses}
+
+Set Up Project
+    [Documentation]    Creates the DS Project (if not exists), creates the data connection for the models,
+    ...                creates caikit runtime. This can be used as test setup
+    [Arguments]    ${namespace}    ${single_prj}=${TRUE}    ${enable_metrics}=${FALSE}    ${dc_name}=kserve-connection
+    Open Model Serving Home Page
+    IF    ${single_prj}
+        Try Opening Create Server
+        Wait for RHODS Dashboard to Load    wait_for_cards=${FALSE}    expected_page=Data Science Projects
```

Comment:
why do we need to handle this case?

Reply:
`Try Opening Create Server` deletes all existing projects to test with only a single DS Project present - This way we can force multiple projects at the same time by specifying `singel_prj=${FALSE}`